### ISSUE TITLE
Ranger Freezes when trying to display an image (Using Kitty with Tmux)

### ISSUE THREAD

**Issue by riceset** [ranger/ranger#2414]
#### Runtime Environment

- Operating system and version: MacOS Big Sur 11.5.1
- Terminal emulator and version: Kitty 0.22.2
- Python version: 3.6.12
- Ranger version/commit: 1.9.3
- Locale: en_US.UTF-8

#### Current Behavior
The application completely freezes when trying to display an image.

#### Possible Solutions
It probably has to do with Tmux because it works normally without it.

#### Steps to reproduce
1. Open kitty
2. Open Tmux
3. Open Ranger
4. Try viewing an image

#### `rc.config` contents
```
set preview_images true
set preview_images_method kitty
```

**Comment by toonn** [comment]
Could you try reproducing this on master? It's probably still an issue but that gives us a recent point of reference.

**Comment by MattiaSilvestri** [comment]
Are there any updates on this? I have the same issue.

**Comment by toonn** [comment]
Can you reproduce it on the master branch?

**Comment by MattiaSilvestri** [comment]
Yes, just tried and I get the same behaviour (ranger version 1.9.3).

**Comment by toonn** [comment]
Thank you. #2846 seems related.

**Comment by hut** [comment]
It should be fixed now. Can you please check again on the master branch @MattiaSilvestri ?
